LondonBritishLibrary/orient/BLorient799.xml

```diff
@@ -60,61 +124,124 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                          <q xml:lang="gez">ኈልቈ፡ ካህናት፡ ዘቅዱስ፡ ሩፋኤል፡ ዘአስተጋብኦሙ፡ ነጉሠ፡ ነገሥት፡ በካፋ።</q>
+                          <note>They are arranged into right, f. 87v, and left, f. 88r, according to the months of the year.</note>
+                          </item>
+
+<item xml:id="a3">
+<locus target="#3r"/>
+<desc type="DonationNote">Deed of gift of <persName ref="PRS5617IyasuII">Iyāsu II</persName> and <persName ref="PRS2682BerhanM">Walatta Giyorgis</persName>
+ to the convent of <placeName ref="INS0330DDQ">Dabra Ḍaḥāy</placeName> near <placeName ref="LOC3577Gondar">Gondar</placeName>.</desc>
+<q xml:lang="gez">ነዋ፡ ወሀብነ፡ ንሕነ፡ አያሱ፡<sic/> ዘስመ፡ መንግሥትነ፡ ብርሃን፡ ሰገድ፡ ወእመነ፡ ንግሥት፡ ወለተ፡ ጊዮርጊስ፡ ብርሃን፡ ሞገሳ፡ ለቤተ፡ ክርስቲያን፡ ቍስቋም፡ እንተ፡ ሕንጽት፡
+በምድረ፡ ላይ፡ ጎደር፡<sic/> ዘተሰምየት፡ ደብረ፡ ፀሐይ፡ አልባሰ፡ ዘሐሪር፡ ወዘርቤታተ፡ ወማዕጠንታተ፡ ወአክሊላተ፡ ዘወርቅ፡ ወዘብሩር፡ ፃሕለ፡ ወጽዋዓ፡ ዘብሩር፡ ወመጻሕፍተ፡ ዘብሉይ፡ ወዘሐዲስ፡
+ወኵሎ፡ ንዋያተ፡ ቅድሳት፡ ከመ፡ ይግበሩ፡ ባሙ፡<sic/> መስዋዕተ፡ ለእግዚአብሔር፡ ለመድኃኒተ፡ ንፍስነ፡ ወለካህናቲሃ፡ ወሀብናሆሙ፡ ሀገረ፡ ዘይሰመይ፡ ባጀና፡ ቀዳሚ፡ ዘነበረ፡ ቦቱ፡ ጋላ፡ ወድኅረ፡
+ዘተሠርዓ፡ ቦቱ፡ ዘዌ፡ ከመ፡ ይኩን፡ ለሲሳዮሙ፡ ወለቁርባንሂ፡ ወሀብነ፡ ሀገረ፡ ዘነበረ፡ በእደ፡ ወይዘሮ፡ መስለ፡ ንትያ፡ ዘስሙ፡ ጉንትር፡ ወዘተዝካረ፡ ንጉሥኒ፡ ሀገረ፡ ዘይሰመይ፡ ከች፡ ወለመካንሂ፡
+ወለጥነ፡ መካነ፡ ዘይሰመይ፡ ቆፋሮች፡ በቅድመ፡ አቡነ፡ <persName ref="PRS6072Krestodo">ክርስተደወሉ፡</persName> ጳጳስ፡ ዘኢትዮጵያ፡ ወበቅድመ፡
+አቡነ፡ <persName ref="PRS12300TaklaHaymanot">ተክለ፡ ሃይማኖት፡</persName> መምኅረ፡ ዘደብረ፡ ሊባኖስ፡ መበቅድመ፡<sic/> ዓቃቤ፡ ሰዓት፡ ወሊቃው፡<sic/> አብያተ፡ ክርስቲያናት፡<gap reason="omitted" resp="PRS10171Wright"></gap></q>
+<date notBefore="1723" notAfter="1755"/>
+</item>
+
+<item xml:id="a4">
+<locus target="#4r"/>
+<desc type="LandGrant">Deed of gift of <persName ref="PRS9153TaklaHa">Takla Haymānot, Ṭǝbab Sagad</persName>, to the priests of
+<placeName ref="LOC2389DabraT">Dabra Ṭǝbab</placeName>, near Gondar.</desc>
+<q xml:lang="gez">በስመ፡ አብ፡ <gap reason="omitted" resp="PRS10171Wright"></gap>ንሕነ፡ ንጉሠ፡ ነገሥት፡ ተክለ፡ ሃይማኖት፡ ዘስመ፡ መንግሥትነ፡ ጥበብ፡ ሰገድ፡ ነዋ፡ አዘዝነ፡
+ወወሀብነ፡ ለካህናተ፡ በአታ፡ ውስተ፡ ቤተ፡ መግደስ፡ ዘትሰመይ፡ ደብረ፡ ጥበብ፡ እንተ፡ ተሐንጸት፡ ጽንፈ፡ ጐንደር፡ ከመ፡ ይኩነነ፡ ለመድኃኒተ፡ ሥጋ፡ ወነፍስ፡ አህጉረ፡ ብዙኃተ፡ ዘውእቶን፡ ምድረ፡ ሠረቀርን፡
+ባሻ፡ ቀመስ፡ ወዓረቢያ፡ ባለአምባ፡ ራስ፡ ቀመስ፡ ወኮሰን፡ ዝፋን፡ ቤት፡ በጅሮንድ፡ ቀመስ፡ ወካልአተ፡ ብዙኃተ፡ አህጉረ። ወእምነ፡ አላንቱ፡ አህጉር፡ ዘበጽሖሙ፡ ለእሉ፡ ካህናት፡ በጅዋና፡ ፪፻፺፪ምድር፡ ተቅራና።
+በካብ፡ ላይ፡  ማርያም፡ ፯፻፲፭ምድር፡ ከቅራና። በብራሄላ፡ ፻፴፪ምድር። <gap reason="omitted" resp="PRS10171Wright"></gap>ወእለ፡ ይመጽኡ፡ እምድኅሬነ፡ ነገሥት፡
+ወመኳንንት፡ ከመ፡ ኢይንሥቱ፡ ዘንተ፡ ሥርዓተ፡ አውገዙ፡ ጳጳስነ፡ አቡነ፡ <persName ref="PRS10450YosabI">ዮሳብ፡</persName> ወመምህረ፡ ደብረ፡ ሊባኖስ፡ እጨጌ፡
+<persName ref="PRS11205TasfaG">ተስፋ፡ ጊዮርጊስ፡</persName> ወኵሎሙ፡ ሊቃውንተ፡ ቤተ፡ ክርስቲያን፡ ወዓቃቤ፡ ሰዓት፡ ተክለ፡
+ሃይማኖት።</q>
+<date notBefore="1769" notAfter="1777"/>
+</item>
+
+<item xml:id="a5">
+<locus target="#5r"></locus>
+  <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+          <placeName ref="INS0101MadhaneAlam"/>.</desc>
+<q xml:lang="gez">ሲኖዶስ፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+</item>
+                        </list>
                         </additions>
-                        
```

Comment:
Please, add as an extra, that the name of the owner has been erased, but his wife's name is there.